core/src/test/java/com/gentics/mesh/test/context/MeshTestContext.java

```diff
@@ -609,6 +608,25 @@ public MeshOptions init(MeshTestSetting settings) throws Exception {
 		default:
 			break;
 		}
+		switch (settings.awsContainer()) {
+			case AWS:
+				break;
```

Comment:
If the AWS part does not really work, I would ask for throwing a NotImplemented here, to early fail the possible usages.